rdf/rdf12/rdf-n-triples/c14n/index.html

```diff
@@ -314,7 +314,7 @@ <h2>
         </dt>
         <dd inlist='true' property='mf:entry' resource='https://w3c.github.io/rdf-tests/rdf/rdf12/rdf-n-triples/c14n#literal_with_2_dquotes' typeof='rdft:TestNTriplesPositiveC14N'>
           <div property='rdfs:comment'>
-            <p>Tests canonicalization of literal with 2 squotes &quot;&quot;&quot;a&quot;&quot;b&quot;&quot;&quot;</p>
+            <p>Tests canonicalization of literal with 2 squotes """a""b"""</p>
```

Comment:
```suggestion
            <p>Tests canonicalization of literal with 2 dquotes """a""b"""</p>
```

Reply:
I'll fix this in the manifest.ttl, which generates this.